Base.rte/Activities/MetaFight.lua

```diff
@@ -8,15 +8,15 @@ function MetaFight:BrainCheck()
 	self:ClearObjectivePoints();
 	-- Keep track of which teams we have set objective points for already, since multiple players can be on the same team
 	local setTeam = { [Activity.TEAM_1] = false, [Activity.TEAM_2] = false, [Activity.TEAM_3] = false, [Activity.TEAM_4] = false };
-	
```

Comment:
You got some sort of auto-formatter or something and didn't notice it had run on this file?
I don't mind the change, good to have stupid whitespace cleaned up, but this is an odd place for it.